### DISCUSSION TITLE
feat: PullToRefresh

### DISCUSSION THREAD

**Discussion by lukedukeus** [microsoft/fluentui-blazor#1674] [Ideas]
# 🙋 Feature Request

It would be cool to have a container thay you could swipe down on to invoke a refresh method.

## 🤔 Expected Behavior

![pull-to-refresh](https://github.com/microsoft/fluentui-blazor/assets/23089287/46e592a5-af17-4cd1-9945-4401ad7ee2f8)
[More Details]( https://learn.microsoft.com/en-us/windows/apps/design/controls/pull-to-refresh)

## 😯 Current Behavior

N/A

## 💁 Possible Solution

I would expect the usage to be something like:
```
<FluentPullToRefresh OnRefresh="OnRefresh" Refreshing="refreshing">
    <FluentNavMenu Title="Weather">
        @foreach (var day in forecast)
        {
            <WeatherTile Forecast="@day" />
        }
    </FluentNavMenu>
</FluentPullToRefresh>

@code
{
    private List<WeatherForcast> forecast = new List<WeatherForcast>();

    private bool refreshing = false;
    
    private void OnRefresh()
    {
        refreshing = true;

        forecast = await weatherService.GetForecast();

        refreshing = false;
    }
}
```

## 🔦 Context

I am trying to add a pull to refresh container as it how most users expect to be able to refresh section of data

## 💻 Examples

See expected behavior

**Comment by vnbaaij** [comment]
Moving this to the discussion area to see if there is interest in such functionality

**Comment by vnbaaij** [comment]
See https://github.com/microsoft/fluentui-blazor/pull/1679
